hoot-core/src/main/cpp/hoot/core/conflate/highway/ImpliedDividedMarker.h

```diff
@@ -72,14 +72,21 @@ class ImpliedDividedMarker : public OsmMapOperation
 
   boost::shared_ptr<OsmMap> markDivided();
 
+  virtual QString getInitStatusMessage()
+  { return "Marking road sections that appear to be divided highways..."; }
+
+  virtual QString getCompletedStatusMessage()
+  { return "Marked " + QString::number(_numMarked) + " road sections as divided highways"; }
+
```

Comment:
Associated print statements to make use of the vis/ops counters.

Reply:
That is a good idea.